daprdocs/content/en/operations/resiliency/policies/retries/retries-overview.md

```diff
@@ -0,0 +1,148 @@
+| `matching.httpStatusCodes` | Optional: a comma-separated string of [HTTP status codes or code ranges to retry](#retry-status-codes). Status codes not listed are not retried.<br/>Valid values: 100-599, [Reference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)<br/>Format: `<code>` or range `<start>-<end>`<br/>Example: "429,501-503"<br/>Default: empty string `""` or field is not set. Retries on all HTTP errors. |
+| `matching.gRPCStatusCodes` | Optional: a comma-separated string of [gRPC status codes or code ranges to retry](#retry-status-codes). Status codes not listed are not retried.<br/>Valid values: 0-16, [Reference](https://grpc.io/docs/guides/status-codes/)<br/>Format: `<code>` or range `<start>-<end>`<br/>Example: "1,501-503"<br/>Default: empty string `""` or field is not set. Retries on all gRPC errors. |
+
+
+## Exponential back-off policy
+
+The exponential back-off window uses the following formula:
+
+```
+BackOffDuration = PreviousBackOffDuration * (Random value from 0.5 to 1.5) * 1.5
+if BackOffDuration > maxInterval {
+  BackoffDuration = maxInterval
+}
+```
+
+## Retry status codes
+
+When applications span multiple services, especially on dynamic environments like Kubernetes, services can disappear for all kinds of reasons and network calls can start hanging. Status codes provide a glimpse into our operations and where they may have failed in production. 
+
+### HTTP
+
+The following table includes some examples of HTTP status codes you may receive and whether you should or should not retry certain operations.
+
+| HTTP Status Code        | Safe to retry?         | Description                  |
+| ----------------------- | ---------------------- | ---------------------------- |
+| 404 Not Found           | ❌ No                  | The resource doesn't exist.  |
+| 400 Bad Request         | ❌ No                  | Your request is invalid.     |
+| 401 Unauthorized        | ❌ No                  | Try getting new credentials. |
+| 503 Service Unavailable | ✅ Yes                 | Service might recover.       |
+| 504 Gateway Timeout     | ✅ Yes                 | Temporary network issue.     |
+
+### gRPC 
+
+The following table includes some examples of gRPC status codes you may receive and whether you should or should not retry certain operations.
+
+| gRPC Status Code          | Safe to retry?          | Description                  |
+| ------------------------- | ----------------------- | ---------------------------- |
+| Code 1 CANCELLED          | ❌ No                   | N/A                          |
+| Code 3 INVALID_ARGUMENT   | ❌ No                   | N/A                          |
+| Code 4 DEADLINE_EXCEEDED  | ✅ Yes                  | Retry with backoff           |
+| Code 5 NOT_FOUND          | ❌ No                   | N/A                          |
+| Code 8 RESOURCE_EXHAUSTED | ✅ Yes                  | Retry with backoff           |
```

Comment:
can you also add this to grpc codes

Code 14 UNAVAILABLE | yes | Retry with backoff